### DISCUSSION TITLE
Add a switch to change between units

### DISCUSSION THREAD

**Discussion by matzeruegge** [time4tea/gopro-dashboard-overlay#94] [Ideas]
Because i am more than happy with the implemented layout except the speed unit in mph. I would a appreciate a switch to modify the units metric / imperial. Than most Europeans can use the software out of the box.

**Comment by time4tea** [comment]
I like this idea!

**Comment by time4tea** [comment]
@matzeruegge this is implemented in 0.82.0 - There are now command line switches to change units.

**Reply by matzeruegge** [reply]
That's great. What a performance.
